app/models/ems_event.rb

```diff
@@ -146,6 +146,34 @@ def self.process_vm_in_event!(event, options = {})
         event[:vm_name] ||= vm.name
       end
     end
+
+    event_type = event[:event_type]
+    # both event types are Rhev specific
+    if (event_type == 'USER_ADD_VM' || event_type == 'USER_ADD_VM_FINISHED_SUCCESS') && event[:vm_or_template_id].nil?
```

Comment:
Yeah, that's right, targeted refresh will only refresh targets that exist in the MIQ DB already.  That means that if the element in the event starting the targeted refresh is a new object, then the target moves up the inventory tree until we find something that exists already and we can select that inventory subtree.